pick.c

```diff
@@ -1041,13 +1063,55 @@ get_key(const char **key)
 	return PRINTABLE;
 }
 
+enum key
+get_vi_key(const char **key)
+{
+	static struct {
+		const char	*str;
+		enum key	 key;
+		int		 vi_mode;
+	}			keys[] = {
+		{ " ",		RIGHT,		1 },
```

Comment:
I would rather see the bindings being defined inside `get_key` and adding a field to the struct indicating that some bindings are only applicable to Vi-mode.